### DISCUSSION TITLE
Get the latest news about Bouncy Castle and other open-source software from Keyfactor

### DISCUSSION THREAD

**Discussion by KarolinHem** [bcgit/bc-java#1856] [Announcements]
The Keyfactor Community newsletter offers the latest product updates and resources from our open-source software EJBCA Community, SignServer Community, and Bouncy Castle. 

Stay up to date on the latest news, software updates, hands-on tutorials, and user workshops on Keyfactor’s open-source software. 

**[Sign up for the newsletter](https://www.bouncycastle.org/#popup-newsletter)**

**Comment by escobarcatalinaortiz-zia** [comment]
Karolin your services are no longer needed 
You are relieved of all duties 
Null and void service as of July 29, 2025
